sharepoint/sharepoint-ps/sharepoint-online/Start-SPOSiteOpticalCharacterRecognitionBackfill.md

```diff
@@ -0,0 +1,83 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Start-SPOSiteOpticalCharacterRecognitionBackfill
+applicable: SharePoint Online
+title: Start-SPOSiteOpticalCharacterRecognitionBackfill
+schema: 2.0.0
+author: FarreltinF
+ms.author: fanyi
+ms.reviewer:
+---
+
+# Start-SPOSiteOpticalCharacterRecognitionBackfill
+
+## SYNOPSIS
+
+> [!NOTE]
+> This feature - Optical Character Recognition (OCR) is a pay-as-you-go feature, triggering this cmdlet will incur cost for your organization
```

Comment:
>This feature - Optical Character Recognition (OCR) is a pay-as-you-go feature, triggering this cmdlet will incur cost for your organization [](http://example.com/codeflow?start=2&length=139)

Optical Character Recognition (OCR) is a pay-as-you-go feature. Running this cmdlet will incur costs for your organization.